dcicutils/env_utils.py

```diff
@@ -234,6 +233,34 @@ def get_mirror_env_from_context(settings, allow_environ=ALLOW_ENVIRON_BY_DEFAULT
         return None
 
 
+def get_standard_mirror_env(envname):
+    """
+    This function knows about the standard mirroring rules and infers a mirror env only from that.
+    (In tha sense, it is not guessing and probably needs to be renamed.)
+    If there is no mirror, it returns None.
+
+    This is not the same as blue_green_mirror_env(envname), which is purely syntactic.
+
+    This is also not the same as get_mirror_env_from_context(...), which infers the mirror from contextual
+    information such as config files and environment variables.
+    """
+    return BEANSTALK_PROD_MIRRORS.get(envname, None)
```

Comment:
Should `envname` be `blue` or `green` in this case? ie: not `data` or `staging` but `mastertest` is okay?

Reply:
I think you're asking for better doc on what kinds of arguments this wants. I can add that.

Reply:
Done.